### DISCUSSION TITLE
DuckDNS alternative question

### DISCUSSION THREAD

**Discussion by saeedesmaili** [coollabsio/coolify#2886] [Service Template Requests]
Probably a noob question.

I've been using the free service of [duckdns](https://www.duckdns.org/) to get a subdomain for my homeassistant instance, as it has an easy to setup [addon](https://www.home-assistant.io/integrations/duckdns/). 

Is there a selfhosted service like duckdns I can use to selfhost via coolify and use it with homeassistant?

**Comment by khalilxg** [comment]
hey i hope you find an applicable production ready path to use duckdns+coolify, if you find some good results plzase comment back

**Reply by saeedesmaili** [reply]
No I haven't found any yet.

**Comment by khalilxg** [comment]
what was the problem
